[2025-06-16T17:30:00.131+0900] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-16T17:30:00.190+0900] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: image_ocr_v2.check_file_exists_branch manual__2025-06-16T08:29:57.337742+00:00 [queued]>
[2025-06-16T17:30:00.221+0900] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: image_ocr_v2.check_file_exists_branch manual__2025-06-16T08:29:57.337742+00:00 [queued]>
[2025-06-16T17:30:00.223+0900] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-06-16T17:30:00.277+0900] {taskinstance.py:2890} INFO - Executing <Task(_BranchPythonDecoratedOperator): check_file_exists_branch> on 2025-06-16 08:29:57.337742+00:00
[2025-06-16T17:30:00.298+0900] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=11506) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-16T17:30:00.305+0900] {standard_task_runner.py:72} INFO - Started process 11508 to run task
[2025-06-16T17:30:00.307+0900] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'image_ocr_v2', 'check_file_exists_branch', 'manual__2025-06-16T08:29:57.337742+00:00', '--job-id', '627', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmpuvlsz1fy']
[2025-06-16T17:30:00.316+0900] {standard_task_runner.py:105} INFO - Job 627: Subtask check_file_exists_branch
[2025-06-16T17:30:00.491+0900] {task_command.py:467} INFO - Running <TaskInstance: image_ocr_v2.check_file_exists_branch manual__2025-06-16T08:29:57.337742+00:00 [running]> on host f6d4228a9eb8
[2025-06-16T17:30:00.814+0900] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='image_ocr_v2' AIRFLOW_CTX_TASK_ID='check_file_exists_branch' AIRFLOW_CTX_EXECUTION_DATE='2025-06-16T08:29:57.337742+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-16T08:29:57.337742+00:00'
[2025-06-16T17:30:00.819+0900] {taskinstance.py:732} INFO - ::endgroup::
[2025-06-16T17:30:00.836+0900] {python.py:240} INFO - Done. Returned value was: get_file_info_list_task
[2025-06-16T17:30:00.838+0900] {branch.py:38} INFO - Branch into get_file_info_list_task
[2025-06-16T17:30:00.840+0900] {skipmixin.py:233} INFO - Following branch ('get_file_info_list_task',)
[2025-06-16T17:30:00.874+0900] {skipmixin.py:281} INFO - Skipping tasks [('end_task', -1)]
[2025-06-16T17:30:00.989+0900] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-06-16T17:30:00.991+0900] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=image_ocr_v2, task_id=check_file_exists_branch, run_id=manual__2025-06-16T08:29:57.337742+00:00, execution_date=20250616T082957, start_date=20250616T083000, end_date=20250616T083000
[2025-06-16T17:30:01.113+0900] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-16T17:30:01.263+0900] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-16T17:30:01.270+0900] {local_task_job_runner.py:245} INFO - ::endgroup::
